src/routes/(console)/organization-[organization]/billing/paymentHistory.svelte

```diff
@@ -45,12 +45,22 @@
     onMount(request);
 
     async function request() {
-        invoiceList = await sdk.forConsole.billing.listInvoices($page.params.organization, [
-            Query.limit(limit),
-            Query.offset(offset),
-            Query.notEqual('from', $organization.billingCurrentInvoiceDate),
-            Query.orderDesc('$createdAt')
-        ]);
+        const unfilteredInvoiceList = await sdk.forConsole.billing.listInvoices(
+            $page.params.organization,
+            [
+                Query.limit(limit),
+                Query.offset(offset),
+                Query.notEqual('from', $organization.billingCurrentInvoiceDate),
+                Query.orderDesc('$createdAt')
+            ]
+        );
+        const filteredInvoices = unfilteredInvoiceList.invoices.filter(
+            (invoice) => invoice.status !== 'pending'
+        );
```

Comment:
We should be able to add an extra query to the fetch request instead of doing the filtering in the frontend right?

Something like `Query.notEqual('status', 'pending')`

Reply:
Changed!